unpackaged/config/community/experiences/Traction_Thrive1/views/customAboutUs.json

```diff
@@ -44,7 +44,7 @@
n developers. As it turns out David Helliwell, one of the founders of Thrive and Greg Malpass, the founder of Traction on Demand, met years before and had remained respected and trusted friends since. David reached out to discuss the challenge with Greg who immediately offered his complete support. Greg wasted no time marshalling all of the necessary resources to tackle the challenge at hand. Within hours the best and brightest from Traction and Thrive were collaborating, virtually and remotely as a team. And the sprint began.</span></p><p class=\"ql-quote-block\"><br></p><p class=\"ql-quote-block\"><span style=\"font-size: 16px; font-family: helvetica;\">Traction on Demand’s partnership and Greg’s alliance and friendship with top executives at Salesforce meant immediate activation of the world’s most trusted and influential technology companies. Rob Aker, CEO of </span><a href=\"https://urldefense.proofpoint.com/v2/url?u=http-3A__Salesforce.org&amp;d=DwMFaQ&amp;c=YHcanBYpGrZxEbl-dEmkXYPwk0jOuY077yg_2uofQn0&amp;r=CInS9h7XmCW4n0rd5s71hGj3sfKfsJH-m0xetfnWzg4&amp;m=-hgsO3WdrCedw1rR8tqNYF2rSDnOg3J_GMBcAfSYyD8&amp;s=jHR52HY8_HRdGacAoHlQo1U_gyJGZgzCFDq1UMiRkR4&amp;e=\" target=\"null\" style=\"font-size: 16px; font-family: helvetica; background-color: rgb(255, 255, 255);\">Salesforce.org</a><span style=\"font-size: 16px; font-family: helvetica;\"> who reports directly to Salesforce founder and CEO Marc Benioff, gave Greg his word that this initiative had every ounce of backing that Salesforce could provide and assembled a significant team chaired by Travis Fabian at Salesforce to support the effort however possible.</span></p><p class=\"ql-quote-block\"><br></p><p class=\"ql-quote-block\"><span style=\"font-size: 16px; font-family: helvetica;\">Salesforce immediately offered free licenses for this initiative. Excited by the potential to help more people, the united team (Thrive, Traction on Demand and Salesforce) began to develop a plan that could scale the impact of this initiative beyond BC or Canada, to global proportion. Knowing that innovation and speed were the only hope to scale in time and that any friction to the platform’s distribution, adaptability or quality would slow it’s adoption, the team together made the choice to move to an open source model. Salesforce brought on their opensource expert team, the same team that built and scaled the Non-Profit Success Platform to hundreds of thousands of NGO’s, to help apply the same scale to the platform.</span></p><p class=\"ql-quote-block\"><br></p><p class=\"ql-quote-block\"><span style=\"font-size: 16px; font-family: helvetica;\">Our goal, all of ours, is to try to help save lives, to preserve the integrity of the healthcare systems and to support our invaluable healthcare workers. We applaud the tireless dedication and courage of all of our health care workers around the globe. It is our sincere hope that this product can make their jobs a little bit better.</span></p></blockquote><p>&nbsp;</p>"
```

Comment:
have these about us links been tested?  there was an issue with them previously that has been fixed.

Reply:
yup, i tested with master branch, images were not displaying
it my mistake of using static src=\"https://sandbox-power-speed-9033-dev-ed-1717d2f45a2.cs45.force.com/TractionThrive/file-asset/LogoTHToDSmall?v=1\"
so the update contains updated url for images and is correct now
and we have to with latest commit changes.